manager/apis/app/v1alpha1/fybrikapplication_types.go

```diff
@@ -10,26 +10,19 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// TargetCatalog contain the specifics for the catalog which needs to be updated.
-// Updates may be adding a new dataset to the catalog (currently supported) or deleting an existing dataset (future)
-type TargetCatalog struct {
-	// CatalogID specifies the catalog where the data will be cataloged.
-	// +optional
-	CatalogID string `json:"catalogID,omitempty"`
-}
-
-// FlowRequirements include the requirements for the data copy operation when there is an explicit copy request
+// FlowRequirements include the requirements specific to the flow
 // Note: Implicit copies done for data plane optimization by Fybrik do not use these parameters
 type FlowRequirements struct {
-	// Catalog indicates that the data asset must be cataloged, and in which catalog to register it.
+	// Catalog indicates that the data asset must be cataloged, and in which catalog to register it
 	// +optional
-	Catalog TargetCatalog `json:"catalog,omitempty"`
+	Catalog string `json:"catalog,omitempty"`
```

Comment:
perhaps TargetCatalogID to avoid confusions?

Reply:
In that case, maybe - `NewCatalogID`? 
`TargetCatalogID` can confuse in the `delete` flow

Reply:
@roytman This field would not be used in the case of delete.  The DataSetID would contain the catalog and datsetid of what needs to be deleted.  I will update the comment appropriately.